tests/integration_tests/performance/test_vhost_user_metrics.py

```diff
@@ -37,7 +37,9 @@ def test_vhost_user_block_metrics(
     # Add a block device to test resizing.
     fs = drive_tools.FilesystemFile(size=orig_size)
     vm.add_vhost_user_drive("scratch", fs.path)
+    vm.add_net_iface()
     vm.start()
+    vm.wait_for_up()
```

Comment:
This is not needed now. It is implicit since ad001192f8061ad10a5a8df0ef14d3d74d3a7dd1

Reply:
removed